doc/config.rst

```diff
@@ -0,0 +1,53 @@
+Configuration File Formats
+============================
+
+Petitboot is capable of parsing multiple flavors of configuration files for gathering the required information
+for booting the next kernel. GRUB2 and SYSLINUX configuration formats are both supported along with a "native"
+format. Documentation for both the `GRUB2 <https://www.gnu.org/software/grub/manual/grub/grub.html#Configuration>`_ 
+and `SYSLINUX <https://wiki.syslinux.org/wiki/index.php?title=Config>`_
+configuration formats can be found in their respective projects documentation.
+
+The native format is made up off multiple "paragraphs", one per boot target, that specify
```

Comment:
I'd suggest a new section heading for the native conf format.

I would say it's made of multiple _sections_, one per boot _option_. Each option begins with a `name` field.